irohad/ordering/impl/on_demand_ordering_service_impl.cpp

```diff
@@ -234,6 +234,14 @@ void OnDemandOrderingServiceImpl::packNextProposals(
   // new reject round
   open_round(
       {round.block_round, currentRejectRoundConsumer(round.reject_round)});
+
+  std::queue<consensus::Round>{}.swap(round_queue_);
```

Comment:
I guess better to move this piece of code to a separate function for better readability.